js/onoffice-prevent-double-form-submission.js

```diff
@@ -0,0 +1,33 @@
+jQuery(document).ready(function ($) {
+    const formElements = getFormsBySpecialHtmlAttribute();
+
+    handleFormSubmissions(formElements);
+
+    function getFormsBySpecialHtmlAttribute() {
+        const formElements = [];
+
+        const forms = document.querySelectorAll('.oo-form, #onoffice-form');
+        formElements.push(...forms);
+
+        return formElements;
```

Comment:
Is the whole function getFormsBySpecialHtmlAttribute really necessary?
Wouldn't it be the same just to use const formElements = document.querySelectorAll('.oo-form, #onoffice-form'); ?

Reply:
Yes, you're right. I will fix it.

Reply:
I fixed it. Please review again for me. Thanks!